bin/verify-google.js

```diff
@@ -2,10 +2,10 @@
 
 var fs = require('fs');
 var path = require('path');
-var argv = require('minimist')(process.argv.slice(2), { string: ['productId', 'packageName', 'receipt', 'keyFile'] });
+var argv = require('minimist')(process.argv.slice(2), { string: ['productId', 'packageName', 'receipt', 'keyFile', 'subscription'] });
```

Comment:
You don't want the string `true`, you want a boolean, right? Minimist has a better way to achieve that.